frontend/src/app/signUp/Organization/OrganizationForm.tsx

```diff
@@ -29,6 +29,7 @@ import {
 } from "./utils";
 
 import "./OrganizationForm.scss";
+import { useDocumentTitle } from "../../utils/hooks";
```

Comment:
Nit: can you group this import with the rest of the file imports on line 21? Github is throwing a warning here otherwise (likely because we're mixing imports from different directory levels). The same thing applies to SignUpGoals.tsx.

Reply:
ooo thank you! will do!